### ISSUE TITLE
Compiling the pro version for android using Fedora

### ISSUE THREAD

**Issue by manaosdeuva** [nesbox/TIC-80#2381]
Hi, i wanna know how to generate a .apk to use the tic80 editor on my phone. I already compiled and installed the pro version on my pc with fedora and it works fine. Is the apk already generated when compiling for linux?

**Comment by sogaiu** [comment]
I think the last time I tried building the `.apk`, I adapted what I saw around [here](https://github.com/nesbox/TIC-80/blob/4b1ec3a52ff3e632e93aeb0f511721d2070ef67f/.github/workflows/build.yml#L576-L614) (though it's likely that the content of the `build.yml` file was a bit different in the past).

**Comment by gabrielsbadenas** [comment]
> I think the last time I tried building the `.apk`, I adapted what I saw around [here](https://github.com/nesbox/TIC-80/blob/4b1ec3a52ff3e632e93aeb0f511721d2070ef67f/.github/workflows/build.yml#L576-L614) (though it's likely that the content of the `build.yml` file was a bit different in the past).

i tried to do that and now i get this error when building. it seems like i need to find where is the ndk installed. now i'm trying to download the ndk from developer.android.com
Unzipping /home/g/.gradle/wrapper/dists/gradle-7.3-bin/ddwl0k7mt9g6ak16i1m905vyv/gradle-7.3-bin.zip to /home/g/.gradle/wrapper/dists/gradle-7.3-bin/ddwl0k7mt9g6ak16i1m905vyv
Set executable permissions for: /home/g/.gradle/wrapper/dists/gradle-7.3-bin/ddwl0k7mt9g6ak16i1m905vyv/gradle-7.3/bin/gradle

Welcome to Gradle 7.3!

Here are the highlights of this release:
 - Easily declare new test suites in Java projects
 - Support for Java 17
 - Support for Scala 3

For more details see https://docs.gradle.org/7.3/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> SDK location not found. Define location with an ANDROID_SDK_ROOT environment variable or by setting the sdk.dir path in your project's local properties file at '/home/g/git/actions-runner/_work/TIC-Ochenta/TIC-Ochenta/build/android/local.properties'.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2m 7s
Error: Process completed with exit code 1.

**Comment by gabrielsbadenas** [comment]
> I think nesbox once mentioned that forking the TIC-80 repository and getting the CI to work with the fork is another method to get building working.
> 
> I haven't tried that approach myself, but may be it's worth an attempt?

ok but how do i do "DBUILD_PRO=On" for android? there's no android pro on build.yml

**Comment by sogaiu** [comment]
I don't know, but possibly [this line](https://github.com/nesbox/TIC-80/blob/4b1ec3a52ff3e632e93aeb0f511721d2070ef67f/build/android/app/build.gradle#L28) might be relevant.

**Comment by gabrielsbadenas** [comment]
> I don't know, but possibly [this line](https://github.com/nesbox/TIC-80/blob/4b1ec3a52ff3e632e93aeb0f511721d2070ef67f/build/android/app/build.gradle#L28) might be relevant.

yes, it worked. i changed Off to On, commit and then let the actions do its thing and now i have the pro version apk. the apk is uploaded as an artifact. i had to fork the repo to make it work. it compiled on github servers without a problem. maybe i'll make my fork private so that nobody can download the apk even tho it's pretty hidden.
